.github/workflows/full.yml

```diff
@@ -60,6 +60,10 @@ jobs:
         run: |
           pip install -r requirements.txt -r requirements/e2e_requirements.txt
 
+      - name: List dependencies
```

Comment:
Isn't it some debug leftover?

Reply:
Yes but I would like to add it for future :wink: